smtp_test.go

```diff
@@ -22,7 +22,9 @@ var (
 
 func TestDialer(t *testing.T) {
 	d := NewPlainDialer(testHost, testPort, "user", "pwd")
+	d.Identity = "test"
```

Comment:
Could you update TestDialerConfig and TestDialerSSLConfig instead of TestDialer and TestDialerSSL?